### PR TITLE
Changed the page for listed items visually using css | Added list of …

### PR DESCRIPTION
**Pull request by evenchowder**
Not related to specific issue - Improved css design of item page

### CODE REVIEW

**Comment by evenchowder** [comment]
Note: The 2 Grid column changes in style.css were accidental you can revert those 2 it doesn't affect the page

File: public/css/style.css [hunk 1]
```diff
@@ -257,10 +284,34 @@
...
 .bg-primary, .btn-primary {
     background-color: #d50032 !important;
 }
 
+.btn-align {
+    display: flex;
+    justify-content: center;
+    align-items: flex-end;
+}
+
```
Comment (An-Owlbear) [inline public/css/style.css:314]: To clarify, what does flex do for this button, i can't see it doing anything differently from block.

File: src/Views/listings/view.phtml [hunk 2]
```diff
@@ -8,31 +8,47 @@ use GroupThr3e\AJExchange\Models\Listing;
...
-    <div class="view-listing-main-image listing-grid-design">
+<div class="view-listing-grid-container mx-auto">
+    <div class="view-listing-main-image">
         <img class="mainImage h-100 w-100" src="https://dummyimage.com/200x200/000/ffffff&text=Listings" alt="">
     </div>
     <div class="view-listing-header listing-grid-design">
-        <h6><?= $listing->getUser()->getFullName() ?>, <?= $listing->getUser()->getOffice()->getAddress() ?></h6>
         <div>
-            <h4><?= htmlspecialchars($listing->getListingName()) ?></h4>
+            <h1> <?=htmlspecialchars($listing->getListingName()) ?></h1>
```
Comment (An-Owlbear) [inline src/Views/listings/view.phtml:17]: Remove the whitespace here and at the start of other tags with similar whitespace

File: src/Views/listings/view.phtml [hunk 3]
```diff
@@ -8,31 +8,47 @@ use GroupThr3e\AJExchange\Models\Listing;
...
+        <h4>Is exchanging for:<span class="emphasise-item-text"> <?= htmlspecialchars($listing->getDesiredItem()) ?></span></h4>
         <?php elseif($listing->getType()=='giveaway'): ?>
-        <h6>Is looking to give away</h6>
+        <h4>Is giving away</h4>
         <?php endif; ?>
-        <button type="submit" class="btn btn-primary w-100 mt-4">Submit</button>
+        <button type="submit" class="btn btn-primary btn-align w-100 mt-4">Submit</button><br>
+        <h4>Tags:</h4>
+        <div class="home-welcome list-div text-center view-listing-tags">
+            <ul style="list-style-type:none; padding: 0;" class="tag-list home-list-design">
```
Comment (An-Owlbear) [inline src/Views/listings/view.phtml:32]: Use list-unstyled bootstrap class instead of style tag

**Review by An-Owlbear — CHANGES_REQUESTED** [review]
Looks great! Just needs a couple changes to make the code a little more readable

File: src/Views/listings/view.phtml [hunk 4]
```diff
@@ -14,22 +14,22 @@ $title = 'listing';
...
         <?php if($listing->getType()=='sell'): ?>
-        <h4 class="zero-margin"><span><?= $listing->getUser()->getFullName() ?></span> is selling for: <br><span class="emphasise-item-text">£<?= htmlspecialchars($listing->getPrice()) ?></span></h4>
+        <h4 class="zero-margin"><span><?= $listing->getUser()->getFullName() ?></span> is selling for: <br><span class="emphasise-item-text">£<?=htmlspecialchars($listing->getPrice()) ?></span></h4>
         <?php elseif($listing->getType()=='exchange'): ?>
-        <h4>Is exchanging for:<span class="emphasise-item-text"> <?= htmlspecialchars($listing->getDesiredItem()) ?></span></h4>
+        <h4>Is exchanging for:<span class="emphasise-item-text"><?=htmlspecialchars($listing->getDesiredItem()) ?></span></h4>
         <?php elseif($listing->getType()=='giveaway'): ?>
         <h4>Is giving away</h4>
         <?php endif; ?>
         <button type="submit" class="btn btn-primary btn-align w-100 mt-4">Submit</button><br>
```
Comment (An-Owlbear) [inline src/Views/listings/view.phtml:29]: The \<br\> should be replaced with a bottom margin. my-4 could be used here

File: public/css/style.css [hunk 5]
```diff
@@ -307,9 +307,7 @@
 }
 
 .btn-align {
```
Comment (An-Owlbear) [inline public/css/style.css:309]: This class is now not needed, as justify-content has no effect unless display is set to flex

File: src/Views/listings/view.phtml [hunk 6]
```diff
@@ -14,22 +14,22 @@ $title = 'listing';
     </div>
     <div class="view-listing-header listing-grid-design">
         <div>
-            <h1> <?=htmlspecialchars($listing->getListingName()) ?></h1>
+            <h1><?=htmlspecialchars($listing->getListingName()) ?></h1>
```
Comment (An-Owlbear) [inline src/Views/listings/view.phtml:17]: Sorry I didn't say earlier, but for consistency all the php tags should have a space at the start and end

**Review by An-Owlbear — CHANGES_REQUESTED** [review]
Thanks! There's just a couple more things that need doing before merging

**Comment by evenchowder** [comment]
To be clear for php syntax did you mean spaces after the <?= or before as I thought you meant before

**Comment by An-Owlbear** [comment]
yeah after the <?=, like <?= $var ?> instead of <?=$var?>

**Review by An-Owlbear — APPROVED** [review]

**Pull request merged**